CHANGELOG.md

```diff
@@ -422,7 +422,7 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 
 ### Bug Fixes
 
-* **cli:** whitelist version env var warnings ([#1057](https://github.com/heroku/cli/issues/1057)) ([e71214d](https://github.com/heroku/cli/commit/e71214d))
+* **cli:** allow list of version env var warnings ([#1057](https://github.com/heroku/cli/issues/1057)) ([e71214d](https://github.com/heroku/cli/commit/e71214d))
```

Comment:
I think the term we've been using is `allowlist` rather than `allow list`.

Reply:
Can we use `allowlist` as a verb? Or maybe I could say "Use allowlist of version env var warnings" or "Filter via allowlist"

Reply:
@chadian My understanding is that `allowlist` would be the substitute of `whitelist` when referring to the product. In this case, and probably in general, the correct replacement would be `allowed list`. What do you think?